src/algebra/big_operators/finprod.lean

```diff
@@ -628,6 +628,22 @@ begin
   exact finprod_mem_congr rfl he₁
 end
 
+/-- The product of `f i` is equal to the product of `g j` if there exists a bijective function
+`e : α → β` such that for all `x` we have `f x = g (e x)`.
+See `finprod_comp`, `fintype.prod_bijective` and `finset.prod_bij` -/
```

Comment:
I think `finset.prod_bij` belongs in the docstring of `finprod_mem_eq_of_bij_on` just above, since both take `bij_on`, right?